server/apiProxy.ts

```diff
@@ -188,6 +188,10 @@ export const authorizationOrCookieHeader = async ({
 			return {
 				Authorization: `Bearer ${req.signedCookies[OAuthAccessTokenCookieName]}`,
 			};
+		case 'user-benefits.' + conf.API_DOMAIN:
+			return {
+				Authorization: `Bearer ${req.signedCookies[OAuthAccessTokenCookieName]}`,
+			};
```

Comment:
these two cases look like they return the same thing, so you could probably combine them like this:

```
case 'members-data-api.' + conf.DOMAIN:
case 'user-benefits.' + conf.API_DOMAIN:
  return {
    Authorization: `Bearer ${req.signedCookies[OAuthAccessTokenCookieName]}`,
  };
```